theano/tensor/nnet/opt.py

```diff
@@ -92,6 +94,7 @@ def local_abstractconv_gradweight_gemm(node):
     if node.op.filter_flip:
```

Comment:
I think we should also add the stack trace on the output of `CorrMM_gradWeights`, the `rval` defined on l. 91.